src/frontend/js/widgets/Dashboard/components/TeacherProfileDashboardSidebar/components/OrganizationLinks/_styles.scss

```diff
@@ -0,0 +1,18 @@
+.dashboard-sidebar {
+  &__organization-section {
+    padding: 5px 0 40px 8px;
+
+    &__title {
+      font-size: 14px;
```

Comment:
Use `rem` please

Reply:
rem is really hard to read and i found px in the code. I assumed we use both.

What about a little mixin: 
```sass
$base-font-size: 16px !default
@function torem($value) {
    @return math.div($value, $base-font-size) * 1rem; 
}
```